.circleci/config.yml

```diff
@@ -41,6 +41,9 @@ commands:
       - restore_cache:
           keys:
             - node-v{{ .Environment.CACHE_VERSION }}-{{ checksum "CACHE_KEY1" }}
+      - run:
+          name: "Install lerna"
+          command: sudo npm install -g lerna
```

Comment:
if this package can be embedded in `package.json` or `package-lock.json` with specific version then we can avoid reinstalling this in each build. I think FE team should be able to help here

Reply:
lerna is already inside package.json and package-lock.json

Reply:
This command will fix `sh: 1: lerna: not found` error(lerna@^3.22.0
) and after applying that we will get this error:
```
./src/components/app.jsx 39 bytes [built] [code generated] [1 error]

ERROR in ./src/components/app.jsx
Module build failed (from ../../node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/plugin-proposal-decorators'
Require stack:
- /home/circleci/project/node_modules/@babel/core/lib/config/files/plugins.js
- /home/circleci/project/node_modules/@babel/core/lib/config/files/index.js
- /home/circleci/project/node_modules/@babel/core/lib/index.js
- /home/circleci/project/node_modules/babel-loader/lib/index.js
- /home/circleci/project/node_modules/webpack/node_modules/loader-runner/lib/loadLoader.js
- /home/circleci/project/node_modules/webpack/node_modules/loader-runner/lib/LoaderRunner.js
- /home/circleci/project/node_modules/webpack/lib/NormalModule.js
- /home/circleci/project/node_modules/webpack/lib/NormalModuleFactory.js
- /home/circleci/project/node_modules/webpack/lib/Compiler.js
- /home/circleci/project/node_modules/webpack/lib/webpack.js
- /home/circleci/project/node_modules/webpack/lib/index.js
- /home/circleci/project/node_modules/webpack-cli/lib/webpack-cli.js
- /home/circleci/project/node_modules/webpack-cli/lib/bootstrap.js
- /home/circleci/project/node_modules/webpack-cli/bin/cli.js
- /home/circleci/project/node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at resolve (/home/circleci/project/node_modules/v8-compile-cache/v8-compile-cache.js:164:23)
    at resolveStandardizedName (/home/circleci/project/node_modules/@babel/core/lib/config/files/plugins.js:102:7)
    at resolvePlugin (/home/circleci/project/node_modules/@babel/core/lib/config/files/plugins.js:46:10)
    at loadPlugin (/home/circleci/project/node_modules/@babel/core/lib/config/files/plugins.js:54:20)
    at loadPlugin.next (<anonymous>)
    at createDescriptor (/home/circleci/project/node_modules/@babel/core/lib/config/config-descriptors.js:179:16)
    at createDescriptor.next (<anonymous>)
    at step (/home/circleci/project/node_modules/gensync/index.js:261:32)
    at evaluateAsync (/home/circleci/project/node_modules/gensync/index.js:291:5)
webpack 5.46.0 compiled with 1 error in 476 ms
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @deriv/p2p@0.7.3 build:travis: `rimraf lib && webpack`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @deriv/p2p@0.7.3 build:travis script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2021-07-27T01_16_19_722Z-debug.log
lerna ERR! npm run build:travis exited 1 in '@deriv/p2p'
lerna ERR! npm run build:travis exited 1 in '@deriv/p2p'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! root@ build:travis: `lerna exec --scope @deriv/shared --scope @deriv/components --scope @deriv/translations  --scope @deriv/cashier --scope @deriv/account --scope @deriv/p2p -- npm run build:travis`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the root@ build:travis script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2021-07-27T01_16_19_736Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! root@ bootstrap: `f () { npm ci && lerna link && lerna bootstrap --ci --hoist --strict && lerna link && npm run build:travis && npm run install:qa ;}; f`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the root@ bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

``` 
So, Better to fix `package.json` related to above error before the next release.

Reply:
yeah, its better to fix `package.json` instead of installing `lerna` in the build time